shared/selene/util/email/email.py

```diff
@@ -29,55 +30,70 @@
 
 
 @dataclass
-class EmailMessage(object):
+class EmailMessage:
+    """Data representation of an email to be sent."""
+
     recipient: str
     sender: str
     subject: str
-    template_file_name: str
+    body: str = None
+    template_file_name: str = None
     template_variables: dict = None
-    content_type: str = 'text/html'
+    content_type: str = "text/html"
+
+    def __post_init__(self):
+        if self.body is not None and self.template_file_name is not None:
+            raise ValueError("Specify body or template file name, not both.")
```

Comment:
This checks if values have been passed for both, do we need to check if both are None?

Reply:
Nice catch.  Fixed.